### PR TITLE
Fs game mode name

### PR DESCRIPTION
**Pull request by fszwedo**
Game Mode name container creation component, with tests and styling

### CODE REVIEW

File: src/app/gameModeName.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+const createGameModeName = (modeName) => {
+    let gameModeNameDiv = document.createElement('div');
```
Comment (juliakozlowska) [inline src/app/gameModeName.js:2]: I would change it to `const`

File: src/app/gameModeName.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    gameModeNameDiv.className = 'gameModeContainer';
+    
+    switch (modeName) {
+        case 'People':
+            gameModeNameDiv.id = 'people';
```
Comment (juliakozlowska) [inline src/app/gameModeName.js:7]: Are you going to use those ids somewhere else later? They seem unnecessary now

File: src/app/gameModeName.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+            gameModeNameDiv.textContent = 'Unknown game mode';
+            throw 'Mode name argument was not passed to the createGameModeName component!'
+    }
+
+    document.body.appendChild(gameModeNameDiv); 
+    return gameModeNameDiv;
+}
+
+let test = 'People';
+createGameModeName(test);
```
Comment (juliakozlowska) [inline src/app/gameModeName.js:29]: This should probably be removed before merging

**Review by juliakozlowska — APPROVED** [review]
Looks good, I just left some suggestions about things that could be improved

File: src/app/gameModeName.js (hunk 1 above)
Comment (fszwedo) [reply src/app/gameModeName.js]: definitely makes sense

File: src/app/gameModeName.js (hunk 2 above)
Comment (fszwedo) [reply src/app/gameModeName.js]: I had some bad experiences with stuff not having an identifier when it was needed, so I've decided to add it as it doesn't hurt to have it there :)

I've removed the id's as they probably won't be needed and are easy to be added

File: src/app/gameModeName.js (hunk 3 above)
Comment (fszwedo) [reply src/app/gameModeName.js]: that's a silly mistake, I was sure I removed that 😞

**Pull request merged**